### ISSUE TITLE
relation mellem governancesystem og personer

### ISSUE THREAD

**Issue by JosefAssad** [JosefAssad/SeMaWi#143]
Nu hvor @lenethystrup sidder og tegner klasser til forvaltning af governancesystemer (den terminologi forvirrer mig men lige meget) er der et spørgsmål omkring hvordan vi modellerer relation mellem en bruger og et governancesystem.

Der er for så vidt jeg kan se to approaches her: en direkte relation, således at man på siden for et governancesystem vælger bruger(e) fra en liste direkte. Det har den fordel at det i dag nok er det organisationen er vant til; den fordel er ikke triviel da den betyder at det bliver på den korte bane nemmere at få brugerne på banen.

Det andet approach er, at et governancesystem tildeles et KLE nummer, og fra KLE emnet tildeles den ansvarlige automatisk. Det approach har fordelen at det er "good organisational design" og skaber rigtig god semantisk sammenhæng med andre typer indhold som systemer og geodata, og øvrige indholdstyper i fremtiden.

Jeg tænker lidt at det egentlig ikke er så farligt at køre med begge approaches midlertidigt. Det værste der kan ske er, at siden for et governancesystem som fx. MySpace viser Josef som den direkte ansvarlige, men siger at den KLE ansvarlige er Elton; så det vil være modsigende, men ville den modsigelse så ikke afspejle virkeligheden til en vis grad?

Jeg synes 100% at målet er, at relation via KLE er målet, men nogle gane er "workarounds" ret nyttige og jeg synes det er et workaround at kunne tildele brugere direkte  indtil rolledefinition via KLE er klar.

Tanker, @lenethystrup og @ClockEnd ?

**Comment by ClockEnd** [comment]
@lenethystrup og jeg har diskuteret det - er ikke enige. Min holdning er, at vi tager udgangspunkt i 'mindste bestanddel' og KLE-tagger dem, derefter følger alle de andre roller som add ons på mindste bestanddel. Systemgovernance er en ting mens indholdsgovernance er noget andet, og jeg vil mene at det er en stor fordel at holde dem adskilt

**Comment by JosefAssad** [comment]
> Systemgovernance er en ting mens indholdsgovernance er noget andet

Yep, enig. Jeg mangler stadig at forstå 100% betydningen af al det her arbejde med governancesystemer/roller, men jeg bliver lidt klogere af for eksempel at læse de sider @lenethystrup opretter under den klasse hun har udviklet.

Er formålet med den klasse, at definere hvilke roller forskellige personer har i forhold til visse systemer? for eksempel, redaktør på facebook, eller systemadministrator på GC2?

Hvis de to eksempler er rigtige så tror jeg jeg kan begynde med at vurdere vores datamodel som helhed og måske komme med nogle gode forslag. :)

**Comment by ClockEnd** [comment]
Altså jeg ser det som mindste bestanddel er en URL (svarende til et datasæt fra GC2) som så kan have diverse roller hæftet på sig, roller som er properties og beskrevt havd indeholder?

**Comment by ClockEnd** [comment]
Så er der systemgovernance, men det bør ligge under Katergori:Systemer. Hvem bestemmer hvilken version, hvilke snitflader osv. Sidernes faglige indhold bestemmes/styres af faglige medarbejdere med et veldefineret KLE

**Comment by ClockEnd** [comment]
På den måde kan vi også bedre udnytte SeMaWis rapport/databsefunktion, fx if overvågning af sider.
Men som sagt @lenethystrup og jeg er langt fra enige

**Comment by JosefAssad** [comment]
Ah, ja. Hvis jeg har forstået rigtigt så er det RIGTIG smart design. fortæl mig om jeg har forstået essensen korrekt:

Et system har i dag personer i de 3 hardkodede roller admin, systemansvarlig, og budgetansvarlig. Det vi skal arbejde hen imod, er at man har prædefinerede roller, så man på en systemside vælger hvilke roller systemet skal bruge, og vælger et KLE emne nummer. Det vil sige, at man IKKE vælger specifikke personer på systemsider.  Så laves der en oversigt som siger "for KLE emne X i rolle Y vil person Z være snvarlig". Og på den måde vil man på en systemside knytte personer til systemet i specifikke roller, men vil også gøre ansvaret mere personuafhængigt ved at opgradere "rolle" begrebet til sin egen klasse og tildele folk KLE/roller i stedet for at knytte dem direkte til systemer.

Er det rigtigt forstået?

**Comment by ClockEnd** [comment]
Umiddelbart ville jeg lade roller if Systemer være en arv fra Kitos 8eller rettere overtage deres rollebegreb) alt den stund at det er Kitos der skal modtage fra SeMaWi på systemsiden - det var det der var udgangspunktet for vores opstart. Og der bør være konkrete personer (med KLE)
